### ISSUE TITLE
less 2.x no longer needed

### ISSUE THREAD

**Issue by levic** [thangngoc89/bs-ant-design#26]
The README suggests to use `less@^2.7.3`.

This should no longer be necessary since the [ant-design incompatibility](https://github.com/ant-design/ant-design/issues/7927) with less 3.x has [been fixed](https://github.com/ant-design/ant-design/pull/13242) since Dec 13 2018 (antd 3.11.3)

**Comment by Enalmada** [comment]
This is probably blocked by https://github.com/thangngoc89/bs-ant-design/issues/25.  However there is a branch where this should be fixed, https://github.com/Enalmada/bs-ant-design/pull/2, so will get that merged in here eventually.
